helm/botkube/values.yaml

```diff
@@ -79,16 +81,18 @@ actions:
 
     # -- Action display name posted in the channels bound to the same source bindings.
     displayName: "Show logs on error"
-    # -- A text value denoting the command run by this action, may contain even based templated values.
-    # The executor is inferred directly from the command, e.g. here we require a kubectl executor
+    # -- Command to execute when the action is triggered. You can use Go template (https://golang.org/pkg/text/template/).
+    # You can use the {{ .Event }}` variable, which contains the event object that triggered the action. See all available event properties on https://github.com/kubeshop/botkube/blob/main/pkg/event/event.go.
+    # The list of helper functions that you can use is available on https://go-task.github.io/slim-sprig/.
```

Comment:
**nit**
-  I just aggregated the info about "template processor" and described it in the same way.
- You have missing ` before {{ .Event }}
```suggestion
    # -- Command to execute when the action is triggered. You can use Go template (https://pkg.go.dev/text/template) together with all helper functions defined by Slim-Sprig library (https://go-task.github.io/slim-sprig).
    # You can use the `{{ .Event }}` variable, which contains the event object that triggered the action. See all available event properties on https://github.com/kubeshop/botkube/blob/main/pkg/event/event.go.
```